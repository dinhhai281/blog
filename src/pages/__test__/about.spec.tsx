import About from '@pages/about';
import { render, layoutMockSetup } from '@utils/testing';
import React from 'react';

beforeAll(() => {
  layoutMockSetup();
});

describe('About', () => {
  it('render correctly', () => {
    const { asFragment } = render(<About></About>);

    expect(asFragment()).toMatchSnapshot();
  });
});
