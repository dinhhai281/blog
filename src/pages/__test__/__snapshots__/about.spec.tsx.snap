// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About render correctly 1`] = `
<DocumentFragment>
  <div
    class="container px-5 pt-12 mx-auto"
  >
    <header
      class="flex justify-between items-center mb-12"
    >
      <h1
        class="text-pink-700 text-6xl font-bold leading-10"
      >
        <a
          href="/en/"
        >
          title
        </a>
      </h1>
    </header>
    <div>
      About page
    </div>
  </div>
</DocumentFragment>
`;
